dev-packages/e2e-tests/test-applications/node-exports-test-app/scripts/consistentExports.ts

```diff
@@ -58,7 +58,12 @@ const DEPENDENTS: Dependent[] = [
     exports: Object.keys(SentryRemix),
   },
   {
-    package: '@sentry/serverless',
+    package: '@sentry/serverless/aws',
+    exports: Object.keys(SentryServerless),
+    ignoreExports: ['cron', 'hapiErrorPlugin', 'enableAnrDetection'],
```

Comment:
I will add another e2e test to validate the subpath exports if we are fine with this approach.